eva-cli/src/main.rs

```diff
@@ -125,18 +125,28 @@ fn dispatch(inputs: &ArgMatches, configuration: &Configuration) -> Result<()> {
             Ok(set_field(configuration, field, id, value)?)
         }
         ("tasks", Some(_submatches)) => {
-            let tasks = block_on(eva::tasks(configuration))?;
-            println!("Tasks:");
-            for task in &tasks {
-                // Indent all lines of task.pretty_print() by two spaces
-                println!("  {}", task.pretty_print().split("\n").join("\n  "));
+            if block_on(eva::num_tasks(configuration))? == 0 {
```

Comment:
I'm really impressed that you put the time and effort into figuring out how the whole stack to the database works!

I have to ask though whether you think doing two database calls is worth it here? We could also look at the list that `eva::tasks` returns and see whether it's empty.

Reply:
You know I am one of those weird individuals who loves databases lol.
It was more of an exercise for me to try using the db system.
Will change this but I can leave the `num_tasks` function in the `libr.rs`, could be useful in the future.

Reply:
Haha, sounds good!